.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "cpputest"]
+	path = cpputest
```

Comment:
Should we put this under tests/cpputest? That way all the unit test stuff is contained

Reply:
Good idea! Went ahead and did that in fd4005d5525a27cdfe3efb85244238a9ff51bfe1